index.html

```diff
@@ -3,13 +3,17 @@
 <div id="app">
   <p>{{ text }}</p>
 
-  <p>
-    <a v-bind:href="linkTo" target="_blank">DuckDuckGo</a>
-  </p>
-
   <p>
     <a :href="linkTo" target="_blank">DuckDuckGo</a>
   </p>
+
+  <p>My name is {{ name }} and I am {{ age }} years old.</p>
+
+  <p>In 5 years I will be {{ addToAge(5) }}.</p>
+
+  <button v-on:click="increaseAge">Age Me</button>
+
+  <button @click="age++">Age Me</button>
```

Comment:
This demonstrates two Vue features.

1. Shorthand syntax, `v-on:click` is equivalent to `@click`.
2. Inline method definition. Note that no `this` is needed.